src/widgets/AppBar.tsx

```diff
@@ -0,0 +1,31 @@
+import * as React from "react";
+import { StackHeaderProps } from "@react-navigation/stack";
+import { Appbar } from "react-native-paper";
+
+import MenuButton from "./MenuButton";
+
+export default function AppBar(props: StackHeaderProps & { menuFallback: boolean }) {
```

Comment:
Why is menuFallback needed? Isn't it always true?

Reply:
No, on split view the view on the right won't need to display the menu.

Reply:
We talked about it before, but I thought we said the split view will be a layout, not a different navigator. So why would the appbar even be called?

Reply:
Because even if there's only one navigator, we still need each view of the layout to have its own Appbar with its custom actions, for example:
- Left View : NoteListScreen [Menu | All notebooks or Title of the selected notebook | Right action menu to sort notes]
- Right View: NoteEditScreen [Title of the Note | Right actions: sync, menu with Edit, Save…]

Reply:
Oh, I didn't realise that we do. Though I think the right appbar would not have the back/left-menu at all, no? So instead of `menuFallback` it should probably be called `hideLeft` or something.

Reply:
I was still seeing the back button on the right view, if we want to close it.

Reply:
You mean you still think that we would need it?

Reply:
Yes, that's what I meant. We won't need the menu button on the right view, but we will probably need the back button. For example to go back from NoteProperties to NoteEdit, or from CollectionEdit/CollectionMembers to CollectionChangelog.